testing/hardhat-deployment/README.md

```
@@ -0,0 +1,49 @@
+- The format of the `repository` should be `github.com/<OrgName/Username>/<Repositoryname>`. Refer to the Kurtosis documentation on [locators](https://docs.kurtosis.com/advanced-concepts/locators) for more information.
+
+- The `script_path` should be relative to the `repository` and `contracts_path` (if present).
+
+- For the wallet, currently only `private_key` is supported.
+
+- For `network`, network name as defined in hardhat.config.ts of the contract repository.
+
+- If the smart contract has prerequisites or dependencies, there are two options:
+    1) If the contracts repository has its dependency built-in, provide the full path to the dependency script.
+    2) Else, to set it locally via kurtosis package. Ensure that the `dependency.sh` file inside the `dependency` folder is completed.
+    
```

Markdownlint

Expected: 0 or 2; Actual: 4
Trailing spaces

(MD009, no-trailing-spaces)